### DISCUSSION TITLE
statistical properties on aspect ratio

### DISCUSSION THREAD

**Discussion by damulaoshi** [neperfepx/neper#286] [Q&A] [answered]
Dear Romain,
Is there any possibility to add some statistical properties on the aspect ratio of grains in 3D?

My command is like:
neper -T:
neper -T -n from_morpho -domain "cube(1,1,1)" -morpho "diameq:lognormal(0.08,0.002),aspratio(2.3,1,1)" -o test1

Here I set the aspect ratio as (2.3,1,1), but I want to change the aspect ratio with some lognormal distributions just like I did for equivalent diameter, is it possible?

Thank you before.
Best regards,
Damu

**Comment by rquey** [comment, marked as answer]
This is not possible with the current implementation, which is based on a global scaling of the tessellation. However, there should be at least some distribution of the aspect ratios of the grains...

What you are requesting would require to define a "grain elongation" metric to pass to `-morpho`.

**Reply by damulaoshi** [reply]
Got it! Thank you so much!
